src/oatpp/parser/json/Utils.cpp

```diff
@@ -476,6 +476,15 @@ std::string Utils::parseStringToStdString(ParsingCaret& caret){
   return "";
   
 }
-    
+
+bool Utils::findDecimalSeparatorInNextWord(const char* data) {
```

Comment:
This method take `ParsingCaret& caret` as a parameter because we are bounded by caret.dataSize and not by '\0' char.